app/views/waste_carriers_engine/cards_forms/new.html.erb

```diff
@@ -12,22 +12,24 @@
       <%= image_tag "example_card.jpg", alt: t(".card_alt_text") %>
     </div>
 
-    <% if @cards_form.errors[:temp_cards].any? %>
-    <div class="form-group form-group-error">
-    <% else %>
-    <div class="form-group">
-    <% end %>
+    <div class="form-group <%="form-group-error" if @cards_form.errors[:temp_cards].any? %>">
```

Comment:
I'm not opposed to the idea of this change as it does make the code more concise – however, as there are very similar lines across all our form views, I'd rather we tackled them all at once as a separate PR. What do you think?

Reply:
Sure why not :) The way I deal with this kind of refactoring is a "fix the file you are changing" approach. Mostly because finding the time for reviewing / refactoring them all at once is usually hard :D

Reply:
Totally get where you're coming from and we do that a lot as well (the Scout Rule!) I think this is one rare occasion where I disagree, just because I know that every form view has got at least one line that looks like this (and the same goes for waste exemptions). We're trying to focus on improving consistency throughout both apps at the moment as then we can do some much heavier refactoring down the line :)

Reply:
Makes sense :)